spec/lucky/action_spec.cr

```diff
@@ -145,6 +164,64 @@ describe Lucky::Action do
       action = PlainText::Index.new(build_context(path: "/?q=test"), params)
       action.params.get(:q).should eq "test"
     end
+
+    it "can get manually defined required params" do
+      action = RequiredParams::Index.new(build_context(path: "/?required_page=1"), params)
+      action.required_page.should eq 1
+    end
+
+    it "raises for missing required params" do
+      action = RequiredParams::Index.new(build_context(path: ""), params)
+      expect_raises(Lucky::Exceptions::MissingParam) { action.required_page }
+    end
```

Comment:
I think it would be good to also add a test for `RequiredParams::Index.with(required_page: 1)` to make sure the argument is added correctly

Reply:
✅

Reply:
Also, `with` doesnt take any named arguments,
```crystal
    def self.with(**args)
      route(**args)
    end

    def self.with(*args)
      route(*args)
    end
```